specification/loadtestservice/LoadTestService.Management/routes.tsp

```diff
@@ -0,0 +1,104 @@
+import "./models.tsp";
+import "./customOperation.tsp";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+
+namespace Microsoft.LoadTestService;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+@@encodedName(LoadTests.createOrUpdate::parameters.resource,
+  "application/json",
+  "LoadTestResource"
+);
+
+@@OpenAPI.extension(LoadTests.createOrUpdate::parameters.resource,
+  "x-ms-client-name",
+  "LoadTestResource"
```

Comment:
There is already resource name `LoadTestResource`. What do you expect C# to generate:
```c#
class LoadTestResource {
   public LoadTestResource  CreateOrUpdate (LoadTestResource : LoadTestResource )    <-- ??? instead of (resource: LoadTestResource)
   {
     ...
   }
}

Reply:
Yes (LoadTestResource : LoadTestResource )    <-- ??? instead of (resource: LoadTestResource)